docs/getting-started/logcli.md

```diff
@@ -64,59 +64,139 @@ Configuration values are considered in the following order (lowest to highest):
-      --key=""           Path to the client certificate key.
+      --cert=""          Path to the client certificate. Can also be set using LOKI_CLIENT_CERT_PATH env var.
+      --key=""           Path to the client certificate key. Can also be set using LOKI_CLIENT_KEY_PATH env var.
+      --org-id=ORG-ID    org ID header to be substituted for auth
       --limit=30         Limit on number of entries to print.
       --since=1h         Lookback window.
       --from=FROM        Start looking for logs at this absolute time (inclusive)
       --to=TO            Stop looking for logs at this absolute time (exclusive)
+      --step=STEP        Query resolution step width
       --forward          Scan forwards through logs.
+      --no-labels        Do not print any labels
+      --exclude-label=EXCLUDE-LABEL ...
+                         Exclude labels given the provided key during output.
+      --include-label=INCLUDE-LABEL ...
+                         Include labels given the provided key during output.
+      --labels-length=0  Set a fixed padding to labels
   -t, --tail             Tail the logs
       --delay-for=0      Delay in tailing by number of seconds to accumulate logs for re-ordering
+
+Args:
+  <query>  eg '{foo="bar",baz=~".*blip"} |~ ".*error.*"'
+
+
+$ logcli help instant-query
+usage: logcli instant-query [<flags>] <query>
+
+Run an instant LogQL query.
+
+This query type can only be used for metrics queries, where the query is
+evaluated for a single point in time. This is equivalent to the Grafana explore
+"table" view; if you want data that is used to build the Grafana graph, you
+should use query instead.
+
+Flags:
+      --help             Show context-sensitive help (also try --help-long and --help-man).
+      --version          Show application version.
+  -q, --quiet            suppress query metadata
+      --stats            show query statistics
+  -o, --output=default   specify output mode [default, raw, jsonl]. raw suppresses log labels and timestamp.
+  -z, --timezone=Local   Specify the timezone to use when formatting output timestamps [Local, UTC]
+      --addr="http://localhost:3100"
+                         Server address. Can also be set using LOKI_ADDR env var.
+      --username=""      Username for HTTP basic auth. Can also be set using LOKI_USERNAME env var.
+      --password=""      Password for HTTP basic auth. Can also be set using LOKI_PASSWORD env var.
+      --ca-cert=""       Path to the server Certificate Authority. Can also be set using LOKI_CA_CERT_PATH env var.
+      --tls-skip-verify  Server certificate TLS skip verify.
+      --cert=""          Path to the client certificate. Can also be set using LOKI_CLIENT_CERT_PATH env var.
+      --key=""           Path to the client certificate key. Can also be set using LOKI_CLIENT_KEY_PATH env var.
+      --org-id=ORG-ID    org ID header to be substituted for auth
+      --limit=30         Limit on number of entries to print.
```

Comment:
does a limit make sense for an instant-query?

Reply:
I had to look at the code, but apparently, yes, limit does make sense for instant-query. `instant-query` with a log line query returns 30 seconds of log lines from the search timestamp. It's not clear why though, @cyriltovena or @owen-d might be able to help explain here.